wdio.conf.js

```diff
@@ -1,12 +1,6 @@
-const path = require('path');
 // eslint-disable-next-line import/no-extraneous-dependencies
-const defaultWdioConfig = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
+const { config } = require('@cerner/terra-functional-testing');
 
-const wdioConfig = defaultWdioConfig.config;
+config.specs = ['./packages/**/wdio/**/*-spec.js'];
```

Comment:
Will this break on windows? We're not using `path.join` anymore to normalize between POSIX vs non-POSIX.

Do we also still need to strip out the `@cerner` stuff from package names?

Reply:
All the other scripts are also unix based and we use devcontainers for windows development anyway so this shouldn't affect it. 

The `@cerner/` stripping is to convert the package name to the folder name, which I am now doing directly using a glob.